students/temu4/DimaTimoschenko02/hw1/app.js

```diff
@@ -0,0 +1,5 @@
+let a = document.createElement('div')
+document.body.appendChild(a)
+a.innerHTML = `
+    <p> text from js file</p>
+`
```

Comment:
Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)